lib/http-bridge/test/integration/Test/Integration/HttpBridge/Scenario/API/Transactions.hs

```diff
@@ -41,6 +42,34 @@ import qualified Network.HTTP.Types.Status as HTTP
 spec :: forall t. (EncodeAddress t, DecodeAddress t) => SpecWith (Context t)
 spec = do
     it "TRANS_CREATE_09 - 0 amount transaction is forbidden on single output tx" $ \ctx -> do
+        (wSrc, payload) <- fixtureZeroAmtSingle ctx
+        r <- request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
+        verify r
+            [ expectResponseCode HTTP.status403
+            , expectErrorMessage errMsg403InvalidTransaction
+            ]
+
+    it "TRANS_CREATE_09 - 0 amount transaction is forbidden on multi-output tx" $ \ctx -> do
+        (wSrc, payload) <- fixtureZeroAmtMulti ctx
+        r <- request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
+        verify r
+            [ expectResponseCode HTTP.status403
+            , expectErrorMessage errMsg403InvalidTransaction
+            ]
+
+    it "TRANS_ESTIMATE_09 - 0 tx fee estimation is allowed? on single output tx" $ \ctx -> do
+        (wSrc, payload) <- fixtureZeroAmtSingle ctx
+        r <- request @ApiFee ctx (postTxFeeEp wSrc) Default payload
+        verify r
+            [ expectResponseCode HTTP.status202 ]
```

Comment:
In case of http-bridge I was rather expecting an error message when trying to estimate fee with `0` amt transaction (the same as it is seen when trying to post transaction with `0` amt -> `expectErrorMessage errMsg403InvalidTransaction`)

@paweljakubas what do you think?

Reply:
@piotr-iohk adding error when any output of transaction is 0 upon estimateFee for http-bridge sounds reasonable - it will inform user that this transaction does not have a chance of passing and 0 amt output should be rectified